LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/IConcentratorDeduplication.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
+{
+    using LoRaWan.NetworkServer.BasicsStation;
+
+    public interface IConcentratorDeduplication
```

Comment:
Is this interface used anywhere?

Reply:
Not yet, will be registered as a singleton after work done in #389 is finished.

Reply:
I will resolve this but feel free to reopen, I am interested in the answer.